packages/themes/src/reset.css

```diff
@@ -4,6 +4,14 @@
   @see https://github.com/tailwindlabs/tailwindcss/blob/3.3/src/css/preflight.css
 */
 
+/* Reset <body> in a layer  */
```

Comment:
This will leak out no?

Reply:
I jinxed myself, put it in!
![image](https://github.com/scalar/scalar/assets/2039539/12efbd6c-9175-4690-8afc-859b79722955)

Reply:
We could put it in the client-app but by nature targeting `<body>` will always leak out of our app.

Reply:
send ittt